src/main/java/gregtech/loaders/recipe/VanillaStandardRecipes.java

```diff
@@ -1033,9 +1033,15 @@ private static void miscRecipes() {
 
         RecipeMaps.ASSEMBLER_RECIPES.recipeBuilder()
                 .input(OrePrefix.plate, Materials.Iron, 3)
-                .input(OrePrefix.ring, Materials.Iron, 4)
+                .input(IRON_MINECART_WHEELS, 2)
                 .outputs(new ItemStack(Items.MINECART))
-                .duration(100).EUt(4).buildAndRegister();
+                .duration(100).EUt(20).buildAndRegister();
+
+        RecipeMaps.ASSEMBLER_RECIPES.recipeBuilder()
+                .input(OrePrefix.plate, Materials.Steel, 3)
+                .input(STEEL_MINECART_WHEELS, 2)
+                .outputs(new ItemStack(Items.MINECART))
+                .duration(100).EUt(20).buildAndRegister();
```

Comment:
These should be put behind a config check for harder iron recipes maybe. Because with that config disabled, there is the vanilla recipe in the crafting table, but our recipe in the assembler. So there is a little clash of difficulty there